packages/mobile/src/screens/app-drawer-screen/left-nav-drawer/LeftNavDrawer.tsx

```diff
@@ -81,17 +130,33 @@ const WrappedLeftNavDrawer = () => {
   const { isEnabled: isFeatureFlagAccessEnabled } = useFeatureFlag(
     FeatureFlags.FEATURE_FLAG_ACCESS
   )
+  const accountUser = useSelector(getAccountUser) as User
+  const { user_id } = accountUser
+  const { tier } = useSelectTierInfo(user_id)
+  const audioBalance = useTotalBalanceWithFallback()
+  const isAudioBalanceLoading = isNullOrUndefined(audioBalance)
+
+  const statuses: Status[] = []
+  const { data: usdcBalance, balanceStatus: usdcBalanceStatus } =
+    useUSDCBalance({
+      isPolling: true,
+      pollingInterval: 3000
+    })
+  if (usdcBalance === null) {
+    statuses.push(usdcBalanceStatus)
+  }
+  const usdcBalanceStatusCombined = combineStatuses(statuses)
+  const isUsdcBalanceLoading =
+    usdcBalance === null || usdcBalanceStatusCombined === Status.LOADING
+  const balanceCents = formatUSDCWeiToFloorCentsNumber(
+    (usdcBalance ?? new BN(0)) as BNUSDC
+  )
+  const usdcBalanceFormatted = formatCurrencyBalance(balanceCents / 100)
```

Comment:
gah need marcus's PR

Reply:
yeah...

Reply:
i assume there's going to be sweeping work that handles all these weird balance formatting